docs/resources/aws_ec2_client_vpn_route.md

```diff
@@ -5,82 +5,82 @@ platform: aws
 
 # aws_ec2_client_vpn_route
 
-Use the `aws_ec2_client_vpn_route` InSpec audit resource to test properties of a single specific AWS EC2 Client VPN route.
+Use the `aws_ec2_client_vpn_route` InSpec audit resource to test the properties of a single specific AWS EC2 client VPN route.
 
-The `AWS::EC2::ClientVpnRoute` specifies a network route to add to a Client VPN endpoint.
+The `AWS::EC2::ClientVpnRoute` specifies a network route to add to a client VPN endpoint.
 
 ## Syntax
 
 Ensure that the client VPN endpoint exists.
 
-    describe aws_ec2_client_vpn_route(client_vpn_endpoint_id: "CLIENT_VPN_ENDPOINT_ID", target_subnet: "TARGET_SUBNET") do
+    describe aws_ec2_client_vpn_route(client_vpn_endpoint_id: "VPN_ID", target_subnet: "SUBNET_ID") do
```

Comment:
The placeholder should be `ENPOINT_ID` (or the original `CLIENT_VPN_ENDPOINT_ID`) since it specifies the endpoint ID of the Client VPN.

```suggestion
    describe aws_ec2_client_vpn_route(client_vpn_endpoint_id: "ENDPOINT_ID", target_subnet: "SUBNET_ID") do
```